src/main/java/net/minecraftforge/fmp/capabilities/MultipartCapabilityHelper.java

```diff
@@ -0,0 +1,175 @@
+package net.minecraftforge.fmp.capabilities;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import net.minecraft.util.EnumFacing;
+import net.minecraftforge.common.capabilities.Capability;
+import net.minecraftforge.common.capabilities.ICapabilityProvider;
+import net.minecraftforge.fmp.multipart.IMultipart;
+import net.minecraftforge.fmp.multipart.IMultipartContainer;
+import net.minecraftforge.fmp.multipart.ISlottedPart;
+import net.minecraftforge.fmp.multipart.PartSlot;
+
+/**
+ * A general use multipart capability helper.
+ */
+public class MultipartCapabilityHelper
```

Comment:
Multiparts do not just have sides. They also have faces. The class allows more specific access for that.